proto/paladin.proto

```diff
@@ -47,13 +47,29 @@ message PaladinTalents {
 
 message RetributionPaladin {
     message Rotation {
-		bool consecration = 1;
+		enum ConsecrateRank {
+			None = 0;
+			Rank1 = 1;
+			Rank4 = 2;
+			Rank6 = 3;
+		}
+		ConsecrateRank consecrateRank = 1;
```

Comment:
Proto fields are named_like_this

Reply:
Implemented all recommended changes. Let me know if anything else stands out or needs to be corrected.